### PR TITLE
docs: streamline tutorials, add and update how-to guides

### PR DESCRIPTION
**Pull request by edibotopic**
UDENG-3595

This PR makes several significant changes to the structure and content of the UP4W tutorial and the how-to sections of the docs.

The tutorial has been greatly simplified and shortened to focus on installation, adding a Pro token and verifying Pro attachment. There are now an increased number of short how-to guides that aim to be brief and reliable sets of instructions for specific tasks.

Tabbed interfaces are used when presenting alternate configuration options.

Major changes:

- **Decoupling of UP4W and Landscape:** installation and automatic Pro-attachment with UP4W does not depend on Landscape. The Getting Started tutorial now only covers the process of installing UP4W, configuring Pro and verifying Pro attachment. The tutorial should now be easier to navigate successfully for a first-time user interested in automatic Pro-attachment of Ubuntu WSL instances. This decoupling is also reflected in the how-to guides, which now treat adding a Pro token and configuring Landscape as independent tasks.
- **Simplification of installation how-to guide:** the original how-to guide on installation and configuration consisted of several dropdown menus, callout blocks and caveats, making it quite difficult to follow for a task that should be straightforward. It has been simplified, in part by presenting a tabbed interface with the two primary options for installing with Windows registry and installing with the graphical Windows application.
- **Extraction of short how-to guides from tutorial content:** another reason for the length of the original tutorial was the inclusion of sections relating to backing up instances and uninstalling applications. These have been extracted into self-contained how-to guides as they contain useful information that a user is likely to return to for a variety of reasons in the context of Ubuntu WSL in general. They are referenced in the tutorial but are not fully included to avoid breaking up the flow of the tutorial.
- **Updated guidelines on Landscape config:** the Landscape integration with UP4W has undergone some changes, particularly with respect to the UI. The documentation now reflects these changes.
- **Increased elaboration of Windows registry option:** using the Windows registry is an important option for enterprise users. It is now given increased prominence in the documentation with some additional text explaining why it is useful and images for user instruction. The getting started tutorial uses the graphical application for installation but the default tab in the how-to guides is for the Windows registry.

Minor changes:

- UI improvements: use of tabbed interfaces for user choice between Windows app and Windows registry
- Internal references: updated internal Sphinx references for many sections
- Rewording/fixes: minor changes throughout
- Updated images: throughout

UDENG-3595

### CODE REVIEW

File: docs/howto/backup-and-restore.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+```text
+PS C:\Users\me\howto> wsl --import ubuntu24.04-b .\backup\ .\backup\Ubuntu-24.04.tar.gz
+PS C:\Users\me\howto> wsl --import ubuntu24.04-c .\backup\ .\backup\Ubuntu-24.04.tar.gz
+```
+
+This will create two additional instances of Ubuntu 24.04 that can be launched and configured independently.
+For example, launching one of these instances is achieved with the command:
+
+```text
+wsl -d ubuntu2404-b
```
Comment (jibel) [inline docs/howto/backup-and-restore.md:67]: There is a dot missing in the name. It shoud be 'ubuntu24.04-b', isn't it?

**Review by jibel — CHANGES_REQUESTED** [review]
Very good. I noticed one typo.

File: docs/howto/backup-and-restore.md (hunk 1 above)
Comment (edibotopic) [reply docs/howto/backup-and-restore.md:67]: You're right, but I think it might be better to just remove the dot in the import statements.

Comment (jibel) [reply docs/howto/backup-and-restore.md:67]: I'm fine with that too.

**Review by CarlosNihelton — CHANGES_REQUESTED** [review]
I like very much the proposed structure, just have few minor items to bring into consideration.

File: docs/howto/set-up-up4w.md [hunk 2]
```diff
@@ -1,267 +1,99 @@
-# How to install and configure UP4W
+# Install UP4W and add a Pro token
 
-## 1. Check that you meet the prerequisites
+<!-- TODO: UP4W app needs to be installed in either case. -->
+<!-- It's where UP4W is configured that matters. -->
```
[the remark this answers is not in the document]
Comment (edibotopic) [reply docs/howto/set-up-up4w.md]: No :blush: 13c621f

File: docs/howto/set-up-landscape-client.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
+# Configure the Landscape client with UP4W
+
+<!-- TODO: assets need to be moved -->
+<!-- example config file needs to be shown -->
+<!-- some minor reworking of tut and refs -->
+<!-- careful not to duplicate content -->
```
[the remark this answers is not in the document]
Comment (edibotopic) [reply docs/howto/set-up-landscape-client.md]: No 😊 13c621f

File: docs/howto/assets/registry-child-window.png
[the remark this answers is not in the document]
Comment (edibotopic) [reply docs/howto/assets/registry-child-window.png:1]: Fixed in 1857905

File: docs/howto/backup-and-restore.md (hunk 1 above)
Comment (edibotopic) [reply docs/howto/backup-and-restore.md]: Fixed in 126613b

File: docs/howto/set-up-landscape-client.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+The path can then be referenced during Landscape configuration in the UP4W Windows app.
+If necessary, the SSL public key can be added after the Windows host has first
+been registered in Landscape.
+
+```
+
+````
+
+(howto::config-schema)=
+## Configuration schema
```
[the remark this answers is not in the document]
Comment (CarlosNihelton) [reply docs/howto/set-up-landscape-client.md]: Great! I prefer that.

**Review by CarlosNihelton — APPROVED** [review]
Very good!

**Review by jibel — APPROVED** [review]
Looks good to me. Thanks for this update!

**Comment by edibotopic** [comment]
Thanks both -- I will mark the remaining conversations as "resolved" now.

**Pull request merged**